docs/chainlink-keepers/supported-networks.html

```diff
@@ -93,5 +96,21 @@
 
 {% endmarkdown %}
 
+{% markdown %}
+### Fantom Mainnet
+
+* **Payment Premium (paymentPremiumPPB):** <span>50</span>% gas premium
+    This premium is applied on top of the gas cost paid by the Keeper to perform your Upkeep
+* **Block Count Per Turn (blockCountPerTurn):** <span>200</span>
```

Comment:
The contract seems to indicate this should be 20. Let's confirm.

Reply:
200 block count per turn is good, thanks for double checking. I just updated contract to match final intended config.l

Reply:
200 block count per turn is good, thanks for double checking. I just updated contract to match final intended config.